redaxo/src/addons/phpmailer/pages/checkmail.php

```diff
@@ -64,6 +68,6 @@
 }
 $fragment = new rex_fragment();
 $fragment->setVar('title', $addon->i18n('checkmail_headline'));
-$fragment->setVar('body', $content, false);
+$fragment->setVar('body', $content.$mailerDebug, false);
```

Comment:
Die Variable kann undefiniert sein, wenn ich das richtig sehe.
Wenn es in den else-block geht hier drüber.

Reply:
sie wird doch in Zeil e48 gesetzt

Reply:
Ja, somit nur im if-teil, nicht im else-teil.